TaskNo31/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using TaskNo31;
+
+internal class Program
+{
+    private static void Main(string[] args)
+    {
+        Console.OutputEncoding = System.Text.Encoding.Unicode;
+
+        Console.WriteLine();
+        Console.WriteLine("Part No. 1");
+        Console.WriteLine();
+
+        int one = 1; int two = 2; int three = 3;
+
+        Metodai suma = new Metodai();
```

Comment:
Čia dar nesimokėme tai gal dėl to šiek tiek netikslu. Užtektų vienos eilutės pvz `Metodai skaiciuotuvas = new Metodai();` ir tuomet `skaiciuotuvas.SumosFunkcija()` ir `skaiciuotuvas.DalmensFunkcija()`.